packages/frontend/src/components/project/DescriptionSection.tsx

```diff
@@ -1,24 +1,29 @@
 import React from 'react'
 
-import { ShieldIcon } from '../icons'
 import { OutLink } from '../OutLink'
 import { Section } from './Section'
+import { WarningBar } from './WarningBar'
 
 export interface DescriptionSectionProps {
   editLink: string
   issueLink: string
   description: string
   warning?: string
+  isVerified?: boolean
 }
 
 export function DescriptionSection(props: DescriptionSectionProps) {
   return (
     <Section title="Description" id="description" className="md:!mt-6">
+      {props.isVerified === false && (
+        <WarningBar
+          text="This project includes unverified contracts."
+          color="red"
+          isCritical={true}
+        />
+      )}
       {props.warning && (
-        <div className="DescriptionSection-Warning">
```

Comment:
Oh, and is the `DescriptionSection-Warning` class used anywhere else? If not, please delete it from the `.scss` file